### DISCUSSION TITLE
Neue MARC Felder

### DISCUSSION THREAD

**Discussion by larspetri** [Larian77/Openbiblio#44] [Q&A] [answered]
Wie legt man "best practice" neue MARC Felder an?
In meinem konkreten Fall soll das Feld "Systematik"  mit aktuell 30 Ausprägungen ergänzt werden.
(GEO - Geographie; LIT - Literatur; JUR - Recht ,...)
Könnte man dieses Feld auch default in der Standardeingabemaske einblenden?

**Comment by Larian77** [comment, marked as answer]
Derzeit müsste man das alles händisch machen, also die Marc-Felder in der Datenbank ergänzen und anschließend diese auch in der Standardmaske anlegen.
Alternativ gibt es im Adminbereich unter Medienarten jeweils den Punkt "MARC Fields", wo man sich für bestimmte Medienarten eigene Marc-Felder definieren kann, habe ich selber aber auch noch nie genutzt, hier hat @LeAchim59 aber glaube schon Erfahrungen gesammelt.
